pos_edit_order_line/static/src/js/EditOrderPopup.js

```diff
@@ -1,9 +1,10 @@
 odoo.define("pos_edit_order_line.EditOrderPopup", function (require) {
     "use strict";
 
-    const {useState} = owl.hooks;
+    const {useState} = owl;
     const AbstractAwaitablePopup = require("point_of_sale.AbstractAwaitablePopup");
     const Registries = require("point_of_sale.Registries");
+    const {_t} = require("web.core");
 
     class EditOrderPopup extends AbstractAwaitablePopup {
         constructor() {
```

Comment:
Its better to use setup() than constructor() 
Same comment for EditOrderLineInput.js
https://www.odoo.com/documentation/16.0/developer/reference/frontend/owl_components.html